lib/openstudio-standards/standards/necb/NECB2011/building_envelope.rb

```diff
@@ -292,6 +292,621 @@ def max_fwdr(hdd)
+      specs[:uo    ] = nil
+      specs[:finish] = :none
+      oAtticFloor    = TBD.genConstruction(model, specs)
+
+      specs          = {}
+      specs[:type  ] = :partition
+      specs[:uo    ] = oWallU
+      iAtticWall     = TBD.genConstruction(model, specs)
+
+      specs          = {}
+      specs[:type  ] = :floor
+      specs[:uo    ] = oFloorU
+      specs[:frame ] = :heavy
+      specs[:finish] = :none
+      specs[:finish] = :heavy if structure == :concrete
+      iAtticFloor    = TBD.genConstruction(model, specs)
+
+      specs          = {}
+      specs[:type  ] = :floor
+      specs[:uo    ] = oRoofU
+      specs[:frame ] = :light
+      specs[:frame ] = :heavy if framing == :wood
+      specs[:clad  ] = :none
+      specs[:clad  ] = :heavy if category == "robust"
+      iAtticRoof     = TBD.genConstruction(model, specs)
+
+      intATTIC = OpenStudio::Model::DefaultSurfaceConstructions.new(model)
+      solATTIC = OpenStudio::Model::DefaultSurfaceConstructions.new(model)
+      extATTIC = OpenStudio::Model::DefaultSurfaceConstructions.new(model)
+      intATTIC.setName("BTAP interior constructions ATTIC")
+      solATTIC.setName("BTAP ground constructions ATTIC")
+      extATTIC.setName("BTAP exterior constructions ATTIC")
+      intATTIC.setWallConstruction(iAtticWall)
+      intATTIC.setFloorConstruction(iAtticFloor)
+      intATTIC.setRoofCeilingConstruction(iAtticRoof)
+      solATTIC.setWallConstruction(gAtticWall)
+      solATTIC.setFloorConstruction(gAtticFloor)
+      solATTIC.setRoofCeilingConstruction(gAtticRoof)
+      extATTIC.setWallConstruction(oAtticWall)
+      extATTIC.setFloorConstruction(oAtticFloor)
+      extATTIC.setRoofCeilingConstruction(oAtticRoof)
+
+      setATTIC = OpenStudio::Model::DefaultConstructionSet.new(model)
+      setATTIC.setName("BTAP construction set ATTIC")
+      setATTIC.setDefaultInteriorSurfaceConstructions(intATTIC)
+      setATTIC.setDefaultGroundContactSurfaceConstructions(solATTIC)
+      setATTIC.setDefaultExteriorSurfaceConstructions(extATTIC)
+
+      attics.each { |attic| attic.setDefaultConstructionSet(setATTIC) }
+    end
+
+    # No change for outdoor-facing surfaces (vs BLDG default construction set).
+    # Treat interzone 'plenum' floors as uninsulated ceiling tiles.
+    unless plenums.empty?
+      specs          = {}
+      specs[:type  ] = :partition
+      specs[:uo    ] = nil
+      specs[:clad  ] = :none
+      specs[:finish] = :none
+      iPlenumFloor   = TBD.genConstruction(model, specs)
+
+      intPLENUM = OpenStudio::Model::DefaultSurfaceConstructions.new(model)
+      intPLENUM.setName("BTAP interior constructions PLENUM")
+      intPLENUM.setFloorConstruction(iPlenumFloor)
+
+      setPLENUM = OpenStudio::Model::DefaultConstructionSet.new(model)
+      setPLENUM.setName("BTAP construction set PLENUM")
+      setPLENUM.setDefaultInteriorSurfaceConstructions(intPLENUM)
+
+      plenums.each { |plenum| plenum.setDefaultConstructionSet(setPLENUM) }
```

Comment:
Same for unoccupied, indirectly-conditioned spaces (like plenums), where just the floor surfaces are set as suspended ceiling tiles.